src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area, Drawable {

```

Comment:
Don't use abstract classes to set behavior for classes. Instead, consider using interfaces to define behaviors and use abstract classes to share common state or methods implementation among related classes. Review the use of abstract classes and interfaces in your design.